cmd/basin/publication.go

```diff
@@ -247,6 +249,11 @@ func newPublicationUploadCommand() *cli.Command {
 				Destination: &secure,
 				Value:       true,
 			},
+			&cli.StringFlag{
+				Name:        "timestamp",
+				Usage:       "The time the file was created (default: current epoch in UTC)",
```

Comment:
@sanderpick did this change and the default is [here](https://github.com/tablelandnetwork/basin-cli/pull/16/files#diff-73777451ff3bdcfcd6ee6970408801e8731a6368a183dfca6d1e1b7df2195fc7R308-R310)

Reply:
okay perfect